base/tools/src/main/java/com/netscape/cmstools/logging/AuditModifyCLI.java

```diff
@@ -100,7 +100,7 @@ public void execute(CommandLine cmd) throws Exception {
                     out.println(line);
                 }
 
-                auditConfig = AuditConfig.fromXML(sw.toString());
+                auditConfig = AuditConfig.fromJSON(sw.toString());
```

Comment:
This is going to affect `pki <subsystem>-audit` commands:
https://github.com/dogtagpki/pki/wiki/PKI-Audit-CLI

@06shalini Is it OK to change this command to use JSON files instead of XML?

Reply:
I created PR #3644 to fix the doc.